### DISCUSSION TITLE
About default DAGs on apache-airflow 2.2.1

### DISCUSSION THREAD

**Discussion by yad1d1** [apache/airflow#22718] [Q&A] [answered]
After installing apache-airflow 2.2.1 on Docker hub, the webui doesn't show the default DAG. What should I do if I want to use these default DAGs?

**Comment by potiuk** [comment, marked as answer]
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#load-examples

**Reply by yad1d1** [reply]
Thank you very much! I solved the problem by moving the dag files from the `/example_dags/` directory to the `/opt/airflow/dags/` directory.

**Reply by potiuk** [reply]
That works too :)
